### ISSUE TITLE
Elections

### ISSUE THREAD

**Issue by jasnell** [nodejs/TSC#259]
It's time to kick off the election for TSC Director and TSC Chair. The nomination period starts now through next Thursday (the 11th). @nodejs/tsc members only, please leave a comment here if you intend to run for 1) TSC Director, 2) TSC Chair, 3) Both. If you'd like, you may nominate another TSC member for the positions. If that happens, the nominated member should respond to accept the nomination. 

We will start the voting process on May 11th

**Comment by jasnell** [comment]
I would like to put my name in the hat for consideration for both the TSC Director and chair positions, but primarily the TSC Director seat (very happy to have someone else take the Chair position ;-) ...)

**Comment by mhdawson** [comment]
I would like to put my name in the hat for consideration for the TSC chair position.  

If there is a need I'm also willing on the Director front (although seems that we already have James raising his hand for that).

**Comment by williamkapke** [comment]
[I would like to make sure everyone feels enabled and encouraged to run for these roles](https://github.com/nodejs/community-committee/issues/31) avoiding/eliminating awkwardness people may feel in doing so. To address this, I have an idea that I've floated past a few of the TSC members with very positive reactions. So, I'm going to try it out:

**I am nominating all members of the TSC** for both the Chair and Director positions. If, for any reason, any member really **does not** want to do the role(s), I ask that you **do not** say it. If you do- you can start a landslide of others also doing the same causing the pool of candidates to shrink revealing those that really want to be in the running... and that eliminates the purpose.

Since the TSC's approved voting methods ask the TSC members to rank their choices from most desired to least, I ask you to wait until you win to decline the position(s)- at which time the votes can be recalculated with that person removed from the list. Rinse & repeat.

Also, I would like to see the votes submitted privately & confidentially. For this, I think we can count on our Linux Foundation resources to help us out. The votes could simply be submitted via email.

Perhaps this process will not be popular across the full TSC- and won't happen. In any case, my nominations still stand.

**Comment by jasnell** [comment]
Ping @nodejs/tsc ... the Nominations close tomorrow. I will send details towards the end of the day tomorrow to begin the voting process.

**Comment by mhdawson** [comment]
@jasnell as I understand it all TSC members are now nominated for both positions.  Are you looking for any other comments/input from us ?

**Comment by jasnell** [comment]
No, just a reminder really. However, if anyone wanted to opt out of the nomination, they could do so.

**Comment by joshgav** [comment]
I'm positive about @williamkapke's suggestion that nominations should be opt-out rather than opt-in, but -1 to not allowing opting out till after the vote, that seems inefficient.

@williamkapke: Without explicit nominations, how will we ask people to describe how they'll approach the role? For comparison, we asked those nominated for the Individual membership to describe their platform.

**Comment by nebrius** [comment]
> I'm positive about @williamkapke's suggestion that nominations should be opt-out rather than opt-in, but -1 to not allowing opting out till after the vote, that seems inefficient.

If we do ranked-choice voting or similar mechanism, I don't _think_ not allowing to opt-out till later will be inefficient. I'd love to hear other people's thoughts though.

> Without explicit nominations, how will we ask people to describe how they'll approach the role? For comparison, we asked those nominated for the Individual membership to describe their platform.

/beginsnark are people describing how they'll approach the role now? /endsnark

More seriously though, I don't think this is really comparable to the individual membership. The only people who can vote in this are TSC members, and we all know each other reasonably well already. Happy to hop on a Skype call if you want to talk through it though.

**Comment by cjihrig** [comment]
Also -1 to not being able to opt out.

**Comment by Fishrock123** [comment]
I am not sure that opting-out afterwards will work properly unless we use some form of ranked voting.

**Comment by mikeal** [comment]
Nominating everyone drastically skews the voting metrics.

I'm not sure what the purpose of nominating people who have not even put in the effort to make a single comment is. The expectations on these roles needs to be much higher, these are basically unpaid jobs, asking that they make the effort of putting themselves forward does not seem like a high bar considering the workload.

**Comment by mcollina** [comment]
I would really thank @rvagg for the awesome work done in these positions.

**Comment by jasnell** [comment]
Ok, so here's the plan: Later on today I will be sending an email out to the TSC members, letting everyone know that they have all been nominated. I'm going to request that any one who wishes to opt out can do so.

@mikeal: do you have specific recommendations on the ballot? I was thinking that an email ballot, with the emails being sent to either you and/or @hackygolucky for counting (if either of you do not mind). With the voting to be completed no later than the 29th and the results tallied up and announced the morning of the 30th. 

Does this work for everyone?

/cc @nodejs/tsc

**Comment by joshgav** [comment]
@mikeal's arguments make sense and I think opt-in would be a better approach long term. OTOH there's no great harm in trying opt-out this round so long as we *can* opt out ;), so +1 to @jasnell's proposal.

@nebrius 

> are people describing how they'll approach the role now?

Perhaps we all should :). For example, I think it would benefit Node if the TSC assumed more technical responsibility and leadership and worked to better support the CTC and WGs. I'm trying to contribute to that vision myself and would vote for someone who shares that view.

**Comment by nebrius** [comment]
> I'm not sure what the purpose of nominating people who have not even put in the effort to make a single comment is. The expectations on these roles needs to be much higher, these are basically unpaid jobs, asking that they make the effort of putting themselves forward does not seem like a high bar considering the workload.

Honestly the reason for taking this approach is purely psychological. We've seen in a number of cases where one person nominates themselves and that causes no one _else_ to nominate themselves. It shifts from being "would I be good for this role?" to "Is it worth it to run against this other person?"

I just had a different idea though. In the future, we could have a private nomination process where everyone submits their candidacy to, say, Mikeal or Tracy, and then all candidates are revealed _at the same time_ after the nomination period ends.

**Comment by williamkapke** [comment]
> I'm not sure what the purpose of nominating people who have not even put in the effort to make a single comment is.

... it is to be neutral/unbiased; that's the entire point here.


> For example, I think it would benefit Node if the TSC assumed more technical responsibility and leadership and worked to better support the CTC and WGs. I'm trying to contribute to that vision myself and would vote for someone who shares that view.

... sounds like you're wanting to be nominated! Done!
<br>
This is just ONE non-confrontational attempt to open the door for others that would like to be in the running. Perfect solution? Probably not! We ARE interested in knowing other ideas to address this problem. Perhaps some are just not concerned with the problem? That's an option too.

Yup - ya'll absolutely can opt out at any time... but, please know that you'll be doing so at the discomfort of others. There really isn't any harm in waiting until you're elected to opt out though. 
...Just sayin.

Please feel free to help brainstorm here: https://github.com/nodejs/community-committee/issues/31
(aka: hopefully I can avoid derail this issue (more?))

**Comment by mikeal** [comment]
> ... it is to be neutral/unbiased; that's the entire point here.

Except the effect is the opposite. Rather than having people on the ballot who are putting themselves forward to do the work you've turned it into a pure popularity contest of the current membership.

**Comment by jasnell** [comment]
@mikeal... we need to get the actual voting started. How do you recommend proceeding?

**Comment by williamkapke** [comment]
> you've turned it into a pure popularity contest

... I guess I have a higher faith in our TSC's members' ability to make the choice. BUT- your offensive _opinion_ has been noted.

**Comment by mikeal** [comment]
@williamkapke It's not about faith in individuals, it's about group incentives. We've been going back and forth on a lot of reforms lately and all of them are about getting same group of people to make behave slightly differently given different incentives and constraints. If these same people always made what you would consider perfect choices we wouldn't need to make any changes at all, we wouldn't even *need* governance.

@jasnell I've setup ballotbin with a ranked voted system for each position. Can you lay out the list of each candidate for each position *and* the email list of all voting TSC members and I can load them all into the app.

**Comment by mikeal** [comment]
@jasnell also, what day/time should the voting close?

**Comment by jasnell** [comment]
Absolutely... Here are the nomination lists. I will send you the email list separately via email.

Voting should be completed by May 29th for tallying on May 30th

**TSC Director Nominees**

Anna Henningsen @addaleax
Ben Noordhuis @bnoordhuis
Bert Belder @piscisaureus
Brian White @mscdex
Bryan Hughes @nebrius
Colin Ihrig @cjihrig
Fedor Indutny @indutny
James Snell @jasnell
Jeremiah Senkpiel @fishrock123
Josh Gavant @joshgav
Michael Dawson @mhdawson
Rod Vagg @rvagg
Shigeki Ohtsu @shigeki
Trevor Norris @trevnorris

**TSC Chair Nominees**

Anna Henningsen @addaleax
Ben Noordhuis @bnoordhuis
Bert Belder @piscisaureus
Brian White @mscdex
Bryan Hughes @nebrius
Colin Ihrig @cjihrig
Fedor Indutny @indutny
James Snell @jasnell
Jeremiah Senkpiel @fishrock123
Josh Gavant @joshgav
Michael Dawson @mhdawson
Rod Vagg @rvagg
Shigeki Ohtsu @shigeki
Trevor Norris @trevnorris

**Comment by mikeal** [comment]
The board meeting is on May 29th so you should close before that. Is there some reason you need more than a week for people to vote?

I thought @rvagg said specifically he didn't want to run?

I would like to note that, without so much as a full discussion, the entire election nomination process has been changed. While it was *possible* to nominate everyone, that was never the intention of the current process, and we have now moved entirely to an "opt out" process. This was never the intention of the current process, which encouraged the nomination of others by members as a way to encourage specific nominations, not to migrate to an entirely different process when a single member wished to change it.

As I am not a voting member I can't formally object to these changes but this sets a concerning precedent.

**Comment by jasnell** [comment]
Ah.. right, May 28th then (I had the dates wrong for the board meeting)

As far as I know, @rvagg has not officially opted out since all TSC members were nominated, effectively putting his name back into the hat.

**Comment by cjihrig** [comment]
I'd like to officially opt out of running for both positions.

**Comment by williamkapke** [comment]
@mikeal As mentioned before, I welcome you to add your **opinions** to the discussion at nodejs/community-committee#31 and I ask that you stop stating your **opinions** as facts.

**Comment by jasnell** [comment]
I would definitely like to state that I'm in agreement with @mikeal on this mass nomination bit. My first initial reaction was positive, but as I thought through it more, it's definitely less than idea for all the reasons that @mikeal states. The challenge, however, is that our charter does not actually define how nominations occur and does not provide for automatically opting anyone out once they've been nominated. As acting chair, there's nothing in there that would allow me to impose a policy one way or the other. Therefore, we're a bit stuck by William's nomination. In fact, the entire process here is so undefined that we're fairly way off track. 

At this point, however, what I want is to have the election rather than continue delaying through disagreements over process. So here's what I propose.... absent a clear documented process for this, I'm going to exercise my authority as chair to ask that anyone currently nominated weigh in on whether they ***accept*** the nomination by End-of-Day GMT on Wednesday, May 17th. The voting will start on May 18th. Only individual @nodejs/tsc members who have explicitly ***Accepted*** the nomination for a particular position will be included on the ballot for that position.

I will send an email out to the TSC members expressing this also.

**Comment by mikeal** [comment]
@williamkapke while I certainly have **opinions** regarding this new process the problem I have with what you did has nothing to do with the pros and cons of different methods, it's the fact that you instituted a new process that you effectively described to the TSC for the first time in a comment in this thread, without so much as a discussion with the TSC. you used your deeper understanding of the current rules *as written* to institute a new system you favored without a vote or formal change to the current process. **this is not ok**. 

there's a process for changing governance and election rules and it has not been followed, it has just been cleverly subverted. the impact of this on the current election process may be minimal but the precedent it sets is much more troubling.

**Comment by indutny** [comment]
I don't intend to run, please remove me from the list of nominees.

**Comment by jasnell** [comment]
@mikeal and @williamkapke ... for the time being, please table continued discussion about the process in this particular thread. I welcome you both to open a new issue in the TSC tracker to discuss possible necessary changes to the voting process. I would like to reserve this thread for coordinating the actual nominations.

**Comment by williamkapke** [comment]
@jasnell I appreciate your approach to attempt to put it on a perceived "on-track" path. However, you are the person that people expressed the discomfort in raising their hand against. This means you asking for people to accept the nominations is in your favor and is a conflict of interest. I really doubt you intend it that way- but it is what it is.

**Comment by mscdex** [comment]
I do not intend to run either.

**Comment by mikeal** [comment]
@williamkapke this is not about **opinions**, it's about **intent**.

If it were the intent of the current process to be opt-out rather than opt-in it would have been written as an opt-out system.

If it were the intent of the current process to be opt-out rather than opt-in you wouldn't have a thread in CommComm expressing your opinions for an alternate to the current process.

You have taken **your opinions** and imposed them on this election by using a part of the process against its original intent. It is quite clever, I have to admit, but to pretend this is all within the original intent of the policies seems like quite a stretch.

**Comment by jasnell** [comment]
Once again, @mikeal and @williamkapke ... Please direct further discussion on process to a separate thread so that this one can be used to track the actual nominations.

**Comment by mikeal** [comment]
There is an option in BallotBin to use "Ranked Choice" or "Plurality." These line up with what the TSC Charter calls out as options (Condorcet or Single Transferable Vote). However, there is no process outlined in the Charter for how to *choose* which option and the prior election used a plurality. But, all of our recent discussions have assumed some sort of Ranked Choice voting system.

I'd like some guidance from the TSC on which method to use as I'm not comfortable making such a substantial determination on my own.

**Comment by jasnell** [comment]
Very good question @mikeal ... I think the model that I've had in mind is [Single Transferable Vote](https://en.wikipedia.org/wiki/Single_transferable_vote#Voting)

**Comment by mikeal** [comment]
@jasnell looking through the repo, the only recent reference I can find to Condorcet was this comment I made (https://github.com/nodejs/TSC/issues/256#issuecomment-297645794) and it seems to be mostly unacknowledged so moving forward with what we have done in prior elections seems appropriate. I think what had me a little confused was that you actually *used* ranked voting for the decision on which month to hold the election :)

**Comment by mikeal** [comment]
Ok, notices for the election ballot should have gone out. @shigeki I think there might have been an issue with yours, email me directly so we can try to resolve it.

**Comment by nebrius** [comment]
@mikeal why were ballots sent out just now? The plan set forth by @jasnell was to send them out on May 17th. This did not give us enough time for TSC members to opt out.

I strongly recommend we recall this ballot.

**Comment by nebrius** [comment]
Also, I'm opting out of this nomination because I was just appointed as the CommComm chairperson. I would like to see more people take on leadership positions in the project, and as such I don't want to hold more than one major leadership role.

**Comment by mikeal** [comment]
Sorry, the ballots should have been for an election that begins on May 17th but that wasn't saved. I was able to change the date but because someone already voted I had to pull back the questions and reset it. Apologies, I'll set this up anew and send out again on the 17th.

**Comment by nebrius** [comment]
Thanks @mikeal

**Comment by isaacs** [comment]
As a Node.js community member, seeing people exploit loopholes in the governance rules of order to circumvent the spirit of those rules serves only to reduce my trust in the foundation.

It's not that either opt-in or opt-out are better in my opinion, and I can certainly see the logic in @williamkapke's suggestion. But there is a forum for those kinds of process changes, and this isn't it.

I suggest that nominating everyone, knowing that several will opt out, is a bad faith exploitation of process, and should be ignored while the charter is amended to explicitly either adopt an opt-out process or close the loophole. I'd be interested in seeing a thoughtful discussion of the trade offs.

**Comment by rvagg** [comment]
I won't be standing at this time.

I wouldn't mind seeing a brief blurb from people that are _wanting_ to stand on why they think they are a good fit and/or what they'd like to get out of it. I guess that's something we'd otherwise miss with an opt-out mechanism. 

One word of warning though, this is not a small workload, and the Director position in particular is going to expose you to a level of politics and background people-work that you may not be prepared for and depending on your personality may be quite taxing. I'd be happy to 1:1 with anyone that wants to get some more details on what it's likely to entail for you.

**Comment by bnoordhuis** [comment]
I'm not running either (except from the tax man.)  I've updated https://github.com/nodejs/TSC/issues/259#issuecomment-301265544.

**Comment by addaleax** [comment]
Me neither.

**Comment by jasnell** [comment]
@nebrius, @addaleax, and @bnoordhuis thank you for updating. The rest of you, please do stop debating the process in this thread, as I have asked a couple times now. We have a process forward for this vote established and that's what we'll be going with. 

 Comments that are not about nominations for this specific vote or are not about the specific logistics of the ballot for this vote will be moderated and collapsed after 24 hours.

**Comment by jasnell** [comment]
FYI.. I am only running for the Director position and opting out of running for Chair. I will post my thoughts per @rvagg's [request](#issuecomment-301292534) separately.

**Comment by isaacs** [comment]
@jasnell Where would you like community members to post comments about the nomination process?

**Comment by jasnell** [comment]
@isaacs https://github.com/nodejs/TSC/issues/259#issuecomment-301271007

**Comment by nebrius** [comment]
@isaacs I'd also say https://github.com/nodejs/community-committee/issues/31

**Comment by mhdawson** [comment]
@jasnell we should probably post our "blurbs" in the same place.  Have you already figured out a good place ?  Otherwise we can probably just open a new issue to capture those.

**Comment by jasnell** [comment]
@mhdawson ... New issue works for me

**Comment by shigeki** [comment]
Sorry, I was off. I opt out both. I've just updated https://github.com/nodejs/TSC/issues/259#issuecomment-301265544.

**Comment by mikeal** [comment]
@nebrius CommComm has autonomy to decide on its own how to handle its election, let's not blur the lines between CommComm and the TSC by merging the discussion.

**Comment by nebrius** [comment]
> CommComm has autonomy to decide on its own how to handle its election, let's not blur the lines between CommComm and the TSC by merging the discussion.

I don't really agree with this. Each committee has the _autonomy_ to handle its own elections, yes, but that doesn't mean they _should_ be different. Each committee is struggling with this same problem right now, and I think it's in our best interest to collaborate with each other to come up with the best solution possible. Let's not reinvent the wheel here.

As prior art on sharing processes, CommComm and the TSC have the autonomy to handle their own CoC and moderation, but we've chosen to go with a unified approach because it reduces confusion and also reduces overhead necessary to administer these things.

**Comment by jasnell** [comment]
@mikeal and @nebrius ... sorry, but that particular conversation should be had in a separate thread. thank you

**Comment by mhdawson** [comment]
Added https://github.com/nodejs/TSC/issues/264 to address @rvagg 's suggestion above https://github.com/nodejs/TSC/issues/259#issuecomment-301292534.

**Comment by trevnorris** [comment]
Please remove me from the lists.

**Comment by Fishrock123** [comment]
I am not running for either position, sorry.

**Comment by joshgav** [comment]
I doubt @piscisaureus is either so we're back where we started :)

**Comment by isaacs** [comment]
<details>
@nebrius @jasnell I hear you.  It is _the nominations here_ that I was trying to comment on, though.  Not "opt-out vs opt-in", but rather, I am concerned about how the current process is being hijacked, and suggest the mass nomination be dismissed for that reason.

It seems like I'm the only one voicing this objection, though.  It's been said and doesn't need to be re-said.  I'll self-destruct this message later today so it's not adding clutter.

As always, feel free to reach out to me at <i@izs.me> if you wanna discuss further.
</details>

**Comment by mhdawson** [comment]
@mikeal should we have received a new ballots ?

**Comment by mikeal** [comment]
@jasnell I want to make sure I get this right, can you give the final list of candidates in each position?

**Comment by jasnell** [comment]
@mikeal, given that we did not receive and opt out or acceptance from Bert, and given the opt outs we did receive... The candidates for each position are: 

Director: @jasnell @mhdawson 
Chair: @mhdawson

**Comment by mikeal** [comment]
Ok, so following this means that @mhdawson is the Chair. We don't have an election when there is only one candidate (https://github.com/nodejs/TSC/blob/master/TSC-Charter.md#section-7-elections).

Based on earlier statements I guess this means that if elected to the board director @mhdawson may step down from one of the seats and if he steps down from the chair we'll just need to have another election.

New ballot is up and a new email should have gone out.

**Comment by jasnell** [comment]
@nodejs/tsc: Not sure if new email notifications went out but the link in the ballot email that was sent out a couple of days ago still works. Please cast your votes!

**Comment by mikeal** [comment]
We're still waiting on 6 people to vote in the election, please get it in this week!!!

**Comment by mikeal** [comment]
What time today should we close the election?

Right now the margin between the two candidates is 2 votes and there are still two people who haven't voted :/

**Comment by jasnell** [comment]
Let's call it at 6pm

**Comment by jasnell** [comment]
Pacific time

**Comment by mikeal** [comment]
Ok, might be a little after 6pm if I'm eating dinner at that time :P

**Comment by mhdawson** [comment]
6pm or there about sounds good to me as well.

**Comment by mikeal** [comment]
James Snell has won the election for Board Chair.

Michael Dawson is still the TSC Chair as he won the prior election.

**Comment by jasnell** [comment]
Thank you @mikeal.
